fe-library-app/src/App.tsx

```diff
@@ -6,23 +6,34 @@ import './App.css'
 import AppRouter from './components/AppRouter/AppRouter'
 import NavigationBar from './components/NavigationBar/NavigationBar'
 import SideBar from './components/SideBar/SideBar'
+import AddModifyBookModal from './modals/AddModifyBookModal'
+import { configureAxiosRequestInterceptors } from './services/ServiceConfig'
 
 function App() {
   const [ sideBarOn, setSideBarOn ] = useState(false)
+  const [ addModifyBookModalOn, setAddModifyBookModalOn ] = useState(false)
 
   const onNavClickHandler = () => {
     setSideBarOn((pervState) => !pervState)
   }
+
+  configureAxiosRequestInterceptors()
+
+  const handleModalFinish = () => {
+    window.location.reload()
+  }
+
   return (
     <BrowserRouter>
       <div className='App'>
         <div className='App-sideBar'>
           <NavigationBar onNavClick={onNavClickHandler} />
-          <SideBar display={sessionStorage.getItem('token') != null} />
+          <SideBar display={sessionStorage.getItem('token') != null} afterChoosingHandle={() => setAddModifyBookModalOn(true)}/>
```

Comment:
Can you create some service for managing token at sessionStorage so you can avoid accessing token directly by key from JSX code?

Reply:
Solved